plugin/src/test/groovy/com/novoda/staticanalysis/internal/detekt/DetektIntegrationTest.groovy

```diff
@@ -14,15 +15,23 @@ import static com.novoda.test.LogsSubject.assertThat
 class DetektIntegrationTest {
```

Comment:
These look like changes in this test but actually it is just that `DetektIntegrationTest` was removed and `DetektNewIntegrationTest` was renamed.